File: 20251118_100Chars.py
** start of main.py **

def one_hundred(chars):
    finalChar = ""

    while len(finalChar) < 100:
        finalChar += chars
    
    finalChar = finalChar[0:100]

    print(finalChar)
    return finalChar

one_hundred("One hundred ")
one_hundred("freeCodeCamp ")

** end of main.py **

File: 20251118_100Chars.js
** start of script.js **

function oneHundred(chars) {
  let finalChar = "";

  while (finalChar.length < 100) {
    finalChar += chars;
  }    
  
  finalChar = finalChar.slice(0, 100);

  console.log(finalChar)
  return finalChar;
}

oneHundred("One hundred ")

** end of script.js **

